packages/@aws-cdk-testing/framework-integ/test/aws-sns/test/integ.sns.ts

```diff
@@ -40,6 +40,9 @@ class SNSInteg extends Stack {
       successFeedbackRole: feedbackRole,
       successFeedbackSampleRate: 50,
     });
+
+    // Can import topic from arn
+    Topic.fromTopicArn(this, 'ImportedTopic', topic.topicArn);
```

Comment:
```suggestion
    const topic2 = new Topic(this, 'MyTopic2', {
      topicName: 'fooTopic2',
      displayName: 'fooDisplayName2',
      masterKey: key,
    });
    const importedTopic = Topic.fromTopicArn(this, 'ImportedTopic', topic2.topicArn);
    
    importedTopic.addLoggingConfig({
      protocol: LoggingProtocol.HTTP,
      failureFeedbackRole: feedbackRole,
      successFeedbackRole: feedbackRole,
      successFeedbackSampleRate: 50,
    });
```
Can you please add an extra topic so that the snapshot will get updated and the pipeline will be green? 
Also, since `fromTopicArn` is calling `fromTopicAttributes`, I think you can remove the change in [integ.sns-fifo.ts](https://github.com/aws/aws-cdk/pull/29542/files#diff-bd9737da387a76c89a9eeb2d47667280f85b2dfb8435ccae19dd8653a44365af)